test/DynamoCoreTests/DSEvaluationViewModelTest.cs

```diff
@@ -115,8 +115,11 @@ public void AssertClassName(string guid, string className)
             string varname = GetVarName(guid);
             var mirror = GetRuntimeMirror(varname);
             Assert.IsNotNull(mirror);
```

Comment:
Yeah I know, I was asking you to move it out of here or get rid of it if it's not required.